app/views/shared/_header.html.haml

```diff
@@ -22,4 +22,4 @@
             %li
               %a(href="#") Account
               %a(href="#") Plan and Billing
-              = link_to 'Sign Out', sign_out_path
+              = link_to 'Sign Out', sign_out_path, method: :delete
```

Comment:
You may run into an issue on Heroku with this. I just dealt with this with another student and for some reason Heroku refused to use anything other than a `GET` route in his app. I am not sure if there was an outside factor as it should have worked fine.